src/pathOrderOptimizer.cpp

```diff
@@ -207,8 +207,8 @@ static inline bool pointsAreCoincident(const Point& a, const Point& b)
 */
 void LineOrderOptimizer::optimize(bool find_chains)
 {
-    int gridSize = 5000; // the size of the cells in the hash grid. TODO
-    SparsePointGridInclusive<unsigned int> line_bucket_grid(gridSize);
+    const int grid_size = 2000; // the size of the cells in the hash grid. TODO
+    SparsePointGridInclusive<unsigned int> line_bucket_grid(grid_size);
```

Comment:
What was the reason to change 5000 to 200 grid_size?

Reply:
It was changed to 2000 not 200.

I wanted to reduce the number of hits returned by getNearbyVals() as it was returning far more than is needed.